dotnet/src/Planners/Planners.Core/Action/ActionPlanner.cs

```diff
@@ -14,12 +14,9 @@
 using Microsoft.SemanticKernel.AI;
 using Microsoft.SemanticKernel.Diagnostics;
 using Microsoft.SemanticKernel.Orchestration;
-using Microsoft.SemanticKernel.Planning.Action;
+using Microsoft.SemanticKernel.Planning;
 
-#pragma warning disable IDE0130
-// ReSharper disable once CheckNamespace - Using NS of Plan
-namespace Microsoft.SemanticKernel.Planning;
-#pragma warning restore IDE0130
+namespace Microsoft.SemanticKernel.Planners.Action;
```

Comment:
Think this should be in `namespace Microsoft.SemanticKernel.Planners` and do same for other planner implementations

Reply:
Fixed